### PR TITLE
Add contact page

### PR DESCRIPTION
**Pull request by kcbreuil**
@asasmith waiting for updated svgs for the logo section but other items are good to review

### CODE REVIEW

File: web/src/components/contactInfo.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            <p>
+                Find us on{' '}
+                <a
+                    className="underline text-brand-5 font-light"
+                    href="https://www.linkedin.com/company/airtec-inc/"
+                >
+                    LinkedIn
+                </a>
+            </p>
+        </div>
```
Comment (asasmith) [inline web/src/components/contactInfo.js:36]: Please remove all of the `<br />` elements and use css for spacing

File: web/src/components/contactForm.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                'form-name': form.getAttribute('name'),
+                ...formData,
+            }),
+        })
+            .then(() => setIsSubmitFormSuccessful(true))
+            .catch((error) => error);
+    };
+
+    return (
+        <div className="bg-brand-1 text-white w-80 md:w-[30rem] p-8 rounded-bl-[2.5rem]">
```
Comment (asasmith) [inline web/src/components/contactForm.js:38]: I don't think this element should align with the center of the page. It should probably be `justify-self: flex-end'

File: web/src/components/contactForm.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                    <textarea
+                        rows="5"
+                        className="mb-4 h-10"
+                        id="message"
+                        name="message"
+                        onChange={handleChange}
+                        required
+                    />
+                    <button
+                        className="bg-white border-2 border-brand-1 text-brand-1 hover:bg-gray-300 hover:border-gray-300 focus:bg-gray-300 focus:border-gray-300 font-semibold leading-tight px-8 py-2 mb-6 rounded-bl-2xl tracking-[.12rem] transition-colors place-self-end justify-self-end uppercase w-[135px]"
```
Comment (asasmith) [inline web/src/components/contactForm.js:93]: do you need `place-self` and `justify-self`?

File: web/src/pages/contact.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+
+import Layout from '../components/layout';
+import Hero from '../components/hero';
+import ContactSection from '../components/contactSection';
+// import LogoSection from '../components/logoSection';
```
Comment (asasmith) [inline web/src/pages/contact.js:8]: Remove comment or add this section

File: web/src/pages/contact.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        hero: {
+            image: heroImage,
+            text: 'Contact',
+        },
+    };
+    return (
+        <Layout>
+            <Hero data={content.hero} />
+            <ContactSection />
+            {/* <LogoSection /> */}
```
Comment (asasmith) [inline web/src/pages/contact.js:22]: Remove comment or add this section

File: web/src/components/logoSection.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function LogoSection() {
+    return <section className="flex">{/* svgs will go here */}</section>;
```
Comment (asasmith) [inline web/src/components/logoSection.js:4]: These will probably need to be images, not svgs. Go ahead and add them now.

**Review by asasmith — CHANGES_REQUESTED** [review]

**Comment by asasmith** [comment]
html
Screen Shot 2021-06-22 at 11 12 48 AM

designs
Screen Shot 2021-06-22 at 11 13 38 AM

Looks like this element is not taking up the full width of the screen (minus padding).

File: web/src/components/contactForm.js [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+                'form-name': form.getAttribute('name'),
+                ...formData,
+            }),
+        })
+            .then(() => setIsSubmitFormSuccessful(true))
+            .catch((error) => error);
+    };
+
+    return (
+        <div className="bg-brand-1 text-white w-full md:w-11/12 p-8 place-self-center rounded-bl-[2.5rem]">
```
Comment (asasmith) [inline web/src/components/contactForm.js:38]: Should this be `place-self-end`?

File: web/src/components/logoSection.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Img from 'gatsby-image';
+
+export default function LogoSection({ images }) {
+    return (
+        <section className="container gap-14 grid grid-cols-1 md:grid-cols-2 place-items-center mb-14">
+            {images.map((image) => {
+                return <Img fixed={image.image.childImageSharp.fixed} />;
```
Comment (asasmith) [inline web/src/components/logoSection.js:9]: Since this `image.image` thing got both of us let's rename the element something that isn't `image`.

**Review by asasmith — CHANGES_REQUESTED** [review]

File: web/src/components/contactForm.js (hunk 7 above)
Comment (kcbreuil) [reply web/src/components/contactForm.js:38]: @asasmith no we want it in the center of its respective div

File: web/src/components/contactForm.js (hunk 3 above)
Comment (kcbreuil) [reply web/src/components/contactForm.js]: @asasmith following up about this question above ^^

**Pull request merged**